handlers/map-tactile-svg/map-svg.py

```diff
@@ -0,0 +1,292 @@
+                                        fill='red',
+                                        stroke_width=1.5,
+                                        stroke='red',
+                                        aria_label=label))
+
+    data = {"graphic": svg.asDataUri()}
+    rendering = {
+        "type_id": "ca.mcgill.a11y.image.renderer.TactileSVG",
+        "description": ("Tactile rendering of map centered at latitude " +
+                        str(contents["coordinates"]["latitude"]) +
+                        " and longitude " +
+                        str(contents["coordinates"]["longitude"])),
+        "data": data}
+    try:
+        validator = jsonschema.Draft7Validator(
+            renderer_schema, resolver=resolver
+        )
+        validator.validate(data)
+    except ValidationError as e:
+        logging.error(e)
+        logging.error("Failed to validate the response renderer!")
+        return jsonify("Failed to validate the response renderer"), 500
+    response = {
+        "request_uuid": contents["request_uuid"],
+        "timestamp": int(time.time()),
+        "renderings": [rendering]
+    }
+    try:
+        validator = jsonschema.Draft7Validator(
+            response_schema, resolver=resolver
+        )
+        validator.validate(response)
+    except ValidationError as e:
+        logging.error("Failed to generate a valid response")
+        logging.error(e)
+        return jsonify("Failed to generate a valid response"), 500
+    logging.debug("Sending response")
+    return response
+
+
+def return_stroke_width(street_type):
+    if (street_type == "primary" or street_type == "secondary"):
+        stroke_width = 7.5
+    elif street_type == "tertiary":
+        stroke_width = 6.5
+    elif street_type == "residential":
+        stroke_width = 4.5
+    elif (street_type == "footway" or street_type == "crossing"):
+        stroke_width = 3.0
+    else:
+        stroke_width = 1.5
+    return stroke_width
+
+
+def getDescriptions(street):
+    description = ""
+    default_attributes = ["street_id", "street_name", "nodes"]
+    if "street_name" not in street:
+        default_attributes.append("street_type")
+    for attr in street:
+        if attr not in default_attributes:
+            match attr:
+                case "oneway":
+                    if street[attr]:
+                        description += "oneway, "
+                    else:
+                        description += "not oneway, "
+                case "lanes":
+                    description += str(street[attr]) + " " + \
+                        attr.replace("_", " ")+", "
+                case _:
+                    description += attr.replace("_", " ") + \
+                        " " + str(street[attr])+", "
+
+    # Remove the last ", "
+    if description == "":
+        return None
+    else:
+        return description[:-2]
+
+
+if __name__ == "__main__":
+    app.run(host="0.0.0.0", port=5000, debug=True)
```

Comment:
Note that different debug port may cause confusion since this does run on port 80.